internal/yunikorn/accumulator.go

```diff
@@ -47,7 +47,7 @@ func (a *accumulator) add(event *si.EventRecord) {
 }
 
 // run is responsible for running the accumulator.
-func (a *accumulator) run(ctx context.Context) error {
+func (a *accumulator) run(ctx context.Context) {
```

Comment:
We should remove this completely. We don't need this anymore since if queue doesn't exist, we should sync partitions